### ISSUE TITLE
Cannot display auction and character at same time

### ISSUE THREAD

**Issue by speedweaver** [Auctionator/Auctionator#1022]
When shopping for upgrades, I cannot open both Auctionator and my character screen at the same time. Makes upgrading incredibly difficult.  I swear it used to let us do this.

Also, when I return back to Auctionator from character view, it removes the levels I set in the filter.  It DOES keep me at the location in the selections (e.g. Cloth/Feet).  But I have to re-enter the levels I want to browse.

**Comment by plusmouse** [comment]
I've had a look.

1. Just tried opening the AH and the character screen at the same time, it works for me. Has your screen resolution changed - sometimes the WoW client hides a window if it thinks there isn't enough space for both of them?
2. Changing tabs on the AH will cause the filters to be reset, but this happens without Auctionator being enabled. Do you have a specific set of actions that cause this with Auctionator enabled (and does it happen if Auctionator is disabled)?

**Comment by plusmouse** [comment]
Update: Found that the character screen can be opened after the AH screen is and it will stay visible, but if it is opened first it gets hidden, including when Auctionator is disabled.

**Comment by speedweaver** [comment]
You are correct. I changed my resolution to be extremely high which shrank the size of the windows and now both fit just fine. Thank you

Sent from my iPhone

On Mar 29, 2021, at 11:43 AM, plusmouse ***@***.***> wrote:

﻿

Update: Found that the character screen can be opened after the AH screen is and it will stay visible, but if it is opened first it gets hidden, including when Auctionator is disabled.
